### PR TITLE
Tweak HTV to render 1d and 0d Funcs properly

### PR DESCRIPTION
**Pull request by steven-johnson**
Add a visualization for the remap LUT in local_laplacian to test/demonstrate. (I verified locally the 0d Funcs also work, but am not including a demonstration.)

### CODE REVIEW

File: apps/local_laplacian/local_laplacian_generator.cpp [hunk 1]
```diff
@@ -165,6 +165,17 @@ class LocalLaplacian : public Halide::Generator<LocalLaplacian> {
             gray.add_trace_tag(cfg.to_trace_tag());
         }
 
+        {
```
Comment (abadams) [inline apps/local_laplacian/local_laplacian_generator.cpp]: I use the local laplacian video in talks - I don't want to make it look weird. Maybe use the tone curve in camera pipe instead?

**Comment by steven-johnson** [comment]
PTAL

File: util/HalideTraceViz.cpp [hunk 2]
```diff
@@ -678,78 +677,73 @@ int run(int argc, char **argv) {
                 fi.stats.observe_load(p);
             }
 
-            // Check the tracing packet contained enough information
-            // given the number of dimensions the user claims this
-            // Func has.
-            assert(p.dimensions >= p.type.lanes * fi.config.strides.size());
-            if (p.dimensions >= p.type.lanes * fi.config.strides.size()) {
-                for (int lane = 0; lane < p.type.lanes; lane++) {
+            // zero- or one-dimensional Funcs can have dimensions < strides
```
Comment (abadams) [inline util/HalideTraceViz.cpp]: I don't understand how this is possible. Shouldn't a zero-d function have zero strides, and a one-d function have one Point?

Comment (steven-johnson) [reply util/HalideTraceViz.cpp]: Strides are provided by the `--stride` flag, so they can be anything. If you don't specify them at all, they default to `{{1,0},{0,1}` (aka size=2). 

(If the strides were provided by the trace instrumentation itself, then your assertion would be correct.)

Comment (steven-johnson) [reply util/HalideTraceViz.cpp]: (I'll add this as a comment since clearly it's possibly confusing)

**Pull request merged**